### PR TITLE
a minimal theme for hexo

### PR DESCRIPTION
**Pull request by Farnaziifz**
- [x] Read the [theme publishing doc](https://hexo.io/docs/themes#Publishing) or [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).

### CODE REVIEW

File: source/_data/themes.yml [hunk 1]
```diff
@@ -2391,3 +2391,14 @@
     - English
     - concise
     - responsive
+- name: Hive
+  description: a minimalist theme for hexo
+  link: https://github.com/farnaziifz/hexo-hive
```
Comment (KayvanMazaheri) [inline source/_data/themes.yml]:
```suggestion
  link: https://github.com/farnaziifz/hive-hexo
```

Comment (Farnaziifz) [reply source/_data/themes.yml]: thanks! its Done!!!

File: source/_data/themes.yml [hunk 2]
```diff
@@ -2393,7 +2393,7 @@
     - responsive
 - name: Hive
   description: a minimalist theme for hexo
-  link: https://github.com/farnaziifz/hexo-hive
+  link: https://github.com/farnaziifz/hive-hexo
   preview: http://51.68.177.152/
```
Comment (YoshinoriN) [inline source/_data/themes.yml]: IMHO specify ip address is not good some point of view.
In fact, other theme owners specify themselves domain or github.io.

I think use github pages is better and fastest way for publish static site, if you hasn't your domain.

* [GitHub pages](https://pages.github.com/)
* [Publishing github pages](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/)

Comment (Farnaziifz) [reply source/_data/themes.yml]: Done !!!

**Comment by YoshinoriN** [comment]
Thanks :thumbsup:

**Pull request merged**